src/ls/CodeActions.mli

```diff
@@ -0,0 +1,4 @@
+val code_actions_of_results :
```

Comment:
In general we try very hard to document things in the .mli. In fact, most of the time for PRs I just look at the .mli.
So here it would be probably better to rename the function to code_actions_of_core_matches
and explain what is the string list second argument (since it's not super clear from the type what it is).

Reply:
I guess the string list is a list of filenames. In theory you could use Common.filename, which is an alias to a string,
but recently we started to switch to more strongly typed file names with Fpath.t.
That requires to use a few extra functions. See also libs/commons/File.mli